### PR TITLE
SDI-345: ✨ Add confirm and complete pages

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: server/routes/addPatient/addPatientCompleted.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ])
+
+    return res.render('pages/addPatient/addPatientCompleted', {
+      prisoner,
+      hospital,
+    })
+  }
+
+  view = async (req: Request, res: Response): Promise<void> => {
+    if (!req.session.newMovePrisonerJourney) return res.render('pages/notFound.njk')
```
Comment (petergphillips) [inline server/routes/addPatient/addPatientCompleted.ts:28]: will change the session variable in next PR, currently reusing the move prisoner one

File: server/routes/addPatient/addPatientCompleted.test.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      ],
+      friendlyName: 'John Smith',
+      prisonerNumber: 'A1234AA',
+    } as PrisonerResultSummary)
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+  it('should load the prisoner move completed page', () => {
```
Comment (mikehalmamoj) [inline server/routes/addPatient/addPatientCompleted.test.ts:112]: The description doesn't match the test...

File: server/routes/addPatient/addPatientCompleted.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  ) {}
+
+  private renderView = async (req: Request, res: Response): Promise<void> => {
+    const prisonerNumber = req.query.prisonerNumber as string
+    const hospitalId = req.query.hospitalId as string
+    const { user } = res.locals
+
+    const [hospital, prisoner] = await Promise.all([
+      this.hospitalSearchService.getHospital(hospitalId, user),
+      this.prisonerSearchService.getPrisonerDetails(prisonerNumber, user),
```
Comment (mikehalmamoj) [inline server/routes/addPatient/addPatientCompleted.ts:18]: I was just wondering if there is any scenario where a user is only allowed to add prisoners from their current caseload? If so should we be searching by caseload here?

**Review by mikehalmamoj — APPROVED** [review]

File: server/routes/addPatient/addPatientCompleted.test.ts (hunk 2 above)
Comment (petergphillips) [reply server/routes/addPatient/addPatientCompleted.test.ts:112]: good spot, will fix in next PR

File: server/routes/addPatient/addPatientCompleted.ts (hunk 3 above)
Comment (petergphillips) [reply server/routes/addPatient/addPatientCompleted.ts:18]: prisoners are `OUT` though

**Pull request merged**